pkcs7_test.go

```diff
@@ -317,7 +317,7 @@ func UnmarshalTestFixture(testPEMBlock string) TestFixture {
 			result.Input = derBlock.Bytes
 		case "CERTIFICATE":
 			result.Certificate, _ = x509.ParseCertificate(derBlock.Bytes)
-		case "PRIVATE KEY":
+		case "PRIVATE KEY", "RSA PRIVATE KEY":
```

Comment:
I've changed the fixtures to be plain `PRIVATE KEY`, so that this case could be removed: https://github.com/smallstep/pkcs7/pull/7.

Reply:
I've commented in #7 that those should be RSA PRIVATE KEY, and I've just noticed that in my previous comment, `#8` does not appear because there's no #8 issue. Fixing that now.

Reply:
I'll revert those changes then. I thought you wanted to get rid of the `RSA PRIVATE KEY`, but it's the other way around?